cocotb/share/makefiles/simulators/Makefile.ius

```diff
@@ -49,8 +49,8 @@ EXTRA_ARGS += $(COMPILE_ARGS)
 EXTRA_ARGS += $(SIM_ARGS)
 EXTRA_ARGS += -licqueue
 
-ifneq ($(ARCH),i686)
-EXTRA_ARGS += -64
+ifneq ($(PYTHON_ARCH),32bit)
```

Comment:
Can we please change that to a positive match on 64bit (ifeq)? It's rather confusing to read it this way. (Also applies for all other uses below.)

Reply:
You should agree with @eric-wieser what you like more ;-) (see above)

Reply:
The idea as for now would be to default to 64 bit in any case that is not 32.

Reply:
What would those cases be? That's why I said "assert" PYTHON_ARCH being either 32bit or 64bit. The code currently is easy to misread, as the if**n**def is easily missed.